extensions/amp-wistia-player/0.1/amp-wistia-player.js

```diff
@@ -76,9 +93,23 @@ class AmpWistiaPlayer extends AMP.BaseElement {
     iframe.src = '//fast.wistia.net/embed/iframe/' + encodeURIComponent(
```

Comment:
Would be cool if this was a configurable option in the amp tag, for local and staging testing, now that I'm thinking about it.

Reply:
I'm very sure that google maintainers ain't going to like that.

Reply:
Mm, good point.